### PR TITLE
Allow timespan to be user-defined

### PR DESCRIPTION
**Pull request by mlarraz**
For those who want increased (or decreased) security.

Addresses https://github.com/mgomes/api_auth/issues/88 and https://github.com/mgomes/api_auth/issues/80

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,6 +183,13 @@ If you want to force the usage of another digest method, you should pass it as a
     ApiAuth.authentic?(signed_request, secret_key, :digest => 'sha256')
 ```
 
+For security, requests dated outside of a certain timespan are considered inauthentic.
+The default span is 15 minutes, but you can override this:
+
+```ruby
+    ApiAuth.authentic?(signed_request, secret_key, :timespan => 1.minute)
```
Comment (kjg) [inline README.md]: `1.minute` is an ActiveSupportism which this gem does not depend on, we might want to change this accordingly as to not give the wrong impression.

Comment (kjg) [inline README.md]: How do we make it clear that "timespan" is not the entire duration as to which the request is valid, but instead the amount of time on each side of the request timestamp.

Comment (mlarraz) [inline README.md]: Fair enough

Comment (mlarraz) [inline README.md]: `time_interval` maybe?

Comment (kjg) [inline README.md]: `offset` makes some sense to me, but then I'm not sure how to incorporate `time` into it.

What are your thoughts on `ApiAuth.authentic?(signed_request, secret_key, :within_time_offset => 60)`

Comment (mlarraz) [inline README.md]: Seems a little verbose to me, but maybe that's ok?

Comment (kjg) [inline README.md]: @mgomes Any thoughts on an appropriate name for this window on either side of the date?

Coming up with a good name is the only thing holding this back for me

Comment (mlarraz) [inline README.md]: That could work. I've also seen the term `clock_skew` used, which seems a little clearer to me. I'll go with whatever you decide

Comment (kjg) [inline README.md]: I'm fine with `clock_skew`

Comment (mlarraz) [inline README.md]: Updated

**Pull request merged**